src/os/bluestore/StupidAllocator.cc

```diff
@@ -14,9 +14,9 @@ StupidAllocator::StupidAllocator(CephContext* cct,
                                  const std::string& name,
                                  int64_t _block_size)
   : Allocator(name), cct(cct), num_free(0),
+    block_size(_block_size),
     free(10),
-    last_alloc(0),
-    block_size(_block_size)
+    last_alloc(0)
```

Comment:
could you initialize `last_alloc` in-class?

Reply:
Done